app/api/v1/endpoints/admin.py

```
@@ -228,7 +240,197 @@ async def configure_system(
+        return {
+            "success": True,
+            "pricing_config": pricing_config,
+            "strategy_pricing": profit_sharing_service.strategy_pricing,
+            "explanation": {
+                "platform_fee": "Percentage of profits users pay as platform fee",
+                "credit_calculation": "Credits = Profit Potential × Platform Fee Percentage",
+                "example": f"$100 profit potential = ${pricing_config.get('platform_fee_percentage', 25):.0f} credits at {pricing_config.get('platform_fee_percentage', 25):.0f}% fee"
+            }
+        }
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)